plugins/main/public/components/common/welcome/components/sca_scan/sca_scan.tsx

```diff
@@ -132,10 +136,14 @@ export const ScaScan = compose(
     }
 
     onClickRow = policy => {
-      localStorage.setItem('scaPolicies', JSON.stringify(this.state.policies));
-      window.location.href = `#/overview?tab=sca&redirectPolicy=${policy.policy_id}`;
-      store.dispatch(updateCurrentAgentData(this.props.agent));
-      this.router.reload();
+      const updatedPolicies = [...this.state.policies, policy];
+      this.setState({ policies: updatedPolicies }, () => {
+        localStorage.setItem(
+          'scaPolicies',
+          JSON.stringify(this.state.policies),
+        );
+        window.location.href = `#/overview?tab=sca&redirectPolicy=${policy.policy_id}`;
```

Comment:
Shouldn't this avoid an app reload using `getCore().application.navigateToApp`?

Reply:
Fixed [c2be781](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6061/commits/c2be781cc43b43268161649bb7db544c35ef644c)